### PR TITLE
Candidates for disposal (Boards)

### PR DESCRIPTION
**Pull request by HOMAG-Boris-Wehrle**

### CODE REVIEW

File: Applications/MaterialAssist/Samples/Authentication/AuthorizationKey01.jpg
Comment (joBueche) [inline Applications/MaterialAssist/Samples/Authentication/AuthorizationKey01.jpg:1]: These screenshots are on intelliDivide. I would change it, if you have already the ones for material

File: Applications/MaterialAssist/Samples/Authentication/Readme.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+To create an instance of the client, you will need both a Subscription Id and a Authorization Key. 
+
+```c#
+// Create new instance of the intelliDivide client:
+            
+var client = new MaterialAssistClient(subscriptionId, authorizationKey);
+``` 
+
+Both can be obtained by subscription administratos following these steps:
```
Comment (joBueche) [inline Applications/MaterialAssist/Samples/Authentication/Readme.md:13]: typo "administrators"

**Review by joBueche — APPROVED** [review]

File: Applications/MaterialAssist/Samples/Authentication/AuthorizationKey01.jpg
Comment (HOMAG-Boris-Wehrle) [reply Applications/MaterialAssist/Samples/Authentication/AuthorizationKey01.jpg:1]: Will get replaced as soon as available.

File: Applications/MaterialAssist/Samples/Authentication/Readme.md (hunk 1 above)
Comment (HOMAG-Boris-Wehrle) [reply Applications/MaterialAssist/Samples/Authentication/Readme.md:13]: fixed

**Review by joBueche — APPROVED** [review]

**Pull request merged**